src/__stories__/centered-carousel-story.tsx

```diff
@@ -44,7 +44,9 @@ export const Default: StoryComponent<Args> = ({numItems, initialActiveItem}) =>
                                     <Text2 regular>Page {bulletsProps.currentIndex}</Text2>
                                 </div>
                                 <div style={centerStyle}>
-                                    {bulletsProps.numPages > 1 && <PageBullets {...bulletsProps} />}
+                                    {(bulletsProps.numPages as number) > 1 && (
+                                        <PageBullets {...bulletsProps} />
+                                    )}
```

Comment:
TS v5 now detects (correctly) this as an error, because the type from `bulletProps` is:

```ts
type PageBulletsProps = {
    currentIndex: number;
    numPages: number | ByBreakpoint<number>;
    onPress?: (index: number) => void;
};
```